src/renderer/components/screens/AccountListSidebar/styles.module.scss

```diff
@@ -0,0 +1,203 @@
+@import '../../../../../scss/variables';
+
+.avatar {
+  --local-avatar-size: 48px;
+  --local-avatar-vertical-margin: 8px;
+  --local-avatar-font-size: 26px;
+
+  position: relative;
+  user-select: none;
+  display: inline-block;
+  font-weight: normal;
+
+  height: var(--local-avatar-size);
+  width: var(--local-avatar-size);
+  margin: var(--local-avatar-vertical-margin) 0;
+
+  div.content,
+  img.content {
+    height: var(--local-avatar-size);
+    width: var(--local-avatar-size);
+    border-radius: 100%;
+    pointer-events: none;
+  }
+
+  div.content {
+    background-color: #505050;
+    line-height: var(--local-avatar-size);
+    object-fit: cover;
+    color: var(--avatarLabelColor);
+    text-align: center;
+    font-size: var(--local-avatar-font-size);
+  }
+
+  img.content {
+    object-fit: cover;
+  }
+}
+
+.accountListSidebar {
+  --accountSidebarBg: #2c2c2c;
+
+  background-color: var(--accountSidebarBg);
+
+  width: 58px;
+
+  flex-shrink: 0;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  --als-avatar-size: 42px;
+  --als-avatar-margin: 4px;
+  --als-active-indicator-color: white;
+  padding-top: 5px;
+
+  .Account {
```

Comment:
We should avoid inherited styles like this, especially when the affected elements are not even part of the component. From here I can't tell how and where `.Account` or even deeper `.avatar` is affected.

Patterns to avoid this are to introduce variants of the sub components you want to style and pass this over as a property (for example `<MyComponent secondary />` etc.), if the styling is too far away, then consider creating a new component. Another, more hacky, but still okay approach would be to give it a `className` and pass over your customized css over like that, though I'd avoid that if it is changing the component a lot

Reply:
account is not inherited or am I missing something? .avatar is lazily cloned because I just copied code, we can copy instead of import the react code as well to make the code prettier, but it is already somewhat independent.

with the new system we can not really overwrite existing classes (or at least I don't know how)

Reply:
> with the new system we can not really overwrite existing classes (or at least I don't know how)

You don't apply inheritance in the new system (which leads to complexity which is hard to follow).

Instead you do what I've proposed here:

> Patterns to avoid this are to introduce variants of the sub components you want to style and pass this over as a property (for example <MyComponent secondary /> etc.), if the styling is too far away, then consider creating a new component. Another, more hacky, but still okay approach would be to give it a className and pass over your customized css over like that, though I'd avoid that if it is changing the component a lot